### PR TITLE
Removes Mixpanel and all of its integrations into our code ✌️

### LINKED ISSUE #7211: Nixpanel
Remove Mixpanel integration.

Nix Mixpanel. Get it?

Related Android issue: https://github.com/wordpress-mobile/WordPress-Android/issues/5907

### PR DESCRIPTION
**Pull request by astralbodies**
Closes #7211 

This removes Mixpanel as an analytics agent and all custom code for integrating it into the app. We were using Mixpanel on occasion to turn off Helpshift so I'll be opening a subsequent issue to address that. I left the comments in the Helpshift utils class referring to Mixpanel with a TODO.

To test:
1. Clean.
2. Build.
3. Run.
4. Test.

Needs review: @koke
cc: @daniloercoli, @maxme

### CODE REVIEW

File: WordPress/Credentials/gencredentials.rb [hunk 1]
```diff
@@ -141,15 +129,14 @@ def print_appbotx_api_key(appbotx_api_key)
 EOF
 end
 
-def print_class(client, secret, pocket, mixpanel_dev, mixpanel_prod, crashlytics, hockeyapp, googleplus, helpshift_api_key, helpshift_domain_name, helpshift_app_id, debugging_key, lookback_token, appbotx_api_key)
+def print_class(client, secret, pocket, crashlytics, hockeyapp, googleplus, helpshift_api_key, helpshift_domain_name, helpshift_app_id, debugging_key, lookback_token, appbotx_api_key)
```
Comment (astralbodies) [inline WordPress/Credentials/gencredentials.rb]: @koke Any way to have hound ignore a particular file?

**Comment by frosty** [comment]
@astralbodies You can ignore Hound for now – it's just using a default config because we haven't yet merged the PR to tell it to use our custom SwiftLint config.

File: WordPress/Credentials/gencredentials.rb (hunk 1 above)
Comment (koke) [inline WordPress/Credentials/gencredentials.rb]: Once #7230 is merged we won't need to 😉 😉

File: WordPress/Classes/Utility/HelpshiftUtils.m [hunk 2]
```diff
@@ -15,7 +14,7 @@
 NSString *const UserDefaultsHelpshiftWasUsed = @"wp_helpshift_used";
 NSString *const HelpshiftUnreadCountUpdatedNotification = @"HelpshiftUnreadCountUpdatedNotification";
 // This delay is required to give some time to Mixpanel to update the remote variable
-CGFloat const HelpshiftFlagCheckDelay = 10.0;
+CGFloat const HelpshiftFlagCheckDelay = 2.0;
```
Comment (koke) [inline WordPress/Classes/Utility/HelpshiftUtils.m]: If we're not actually going to check anything, why wait 2 seconds?

Comment (astralbodies) [inline WordPress/Classes/Utility/HelpshiftUtils.m]: Only because I didn't want to remove the mechanism but also didn't want it to happen too quickly in case I'm breaking something. I'm going to do another PR soon to replace this mechanism. ;)
